piezo_web_app/PiezoWebApp/src/handlers/get_logs.py

```diff
@@ -0,0 +1,19 @@
+from tornado_json import schema
+
+from PiezoWebApp.src.handlers.base_handler import BaseHandler
+from PiezoWebApp.src.handlers.schema.schema_helpers import create_object_schema_with_string_properties
+
+
+class GetLogsHandler(BaseHandler):
+    @schema.validate(
+        input_schema=create_object_schema_with_string_properties(['driver_name', 'namespace'], required=['driver_name', 'namespace']),
+        input_example={
+            'driver_name': 'example-driver',
+            'namespace': 'example-namespace'
+        }
```

Comment:
By default all spark applications are run in namespace `default`. This can be configured in the creation of the spark application but they must run in a namespace that already exists. For monitoring it is required that we know the namespace so I think it is likely that we won't want to allow the user to set the namespace for the application and force them to use `default`

Reply:
` [a713e62](/ukaea/piezo/pull/25/commits/a713e62fa847f948e78c927075ee21f5d3306a06)`